Directory.Build.props

```diff
@@ -4,19 +4,16 @@
 
     <PropertyGroup>
         <OutputType>Exe</OutputType>
-        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <TargetFramework>net6.0</TargetFramework>
         <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
     </PropertyGroup>
 
     <PropertyGroup>
-        <NoWarn>$(NoWarn),SA0001,CA1014</NoWarn>
+        <NoWarn>$(NoWarn),SA0001,SA1200,SA1516,CA1014,CA1812</NoWarn>
```

Comment:
SA1516 is a [false positive](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3351)